src/python/graplctl/graplctl/aws/lib.py

```diff
@@ -0,0 +1,125 @@
+import base64
```

Comment:
thank you for using my "commands / lib" pattern lol

Reply:
👍